### ISSUE TITLE
reddestdream.github.io: Bad format for PiHole

### ISSUE THREAD

**Issue by beerisgood** [WaLLy3K/wally3k.github.io#80]
I wonder why https://reddestdream.github.io/Projects/MinimalHosts/etc/MinimalHostsBlocker/minimalhosts is marked with a tick cause that list use a strange format:

```
#MHB73
127.0.0.1		localhost
255.255.255.255 broadcasthost
::1				localhost

:: admob.biz
:: admob.co.kr
0 1app.blob.core.windows.net
0 2912a.v.fwmrm.net
```

Take a look at the zero before the domain and also "::" stand for a comment but a lot of domains have that
Just a few of them.

So i vote for remove that list or mark it at least with cross

**Comment by beerisgood** [comment]
Also related to https://github.com/WaLLy3K/wally3k.github.io/issues/78#issue-553849934

**Comment by Aeriem** [comment]
I believe `::` is the IPv6 equivalent of `0.0.0.0`. ([source](https://en.wikipedia.org/wiki/0.0.0.0#In_IPv6))
`0.0.0.0` is used in place of `127.0.0.1` in some hosts files to reduce latency, avoid interferences with the local server (should you have one) and slowdowns in Windows 8.1. However, it might not be compatible with all systems. (sources: [[1]](https://github.com/StevenBlack/hosts#we-recommend-using-0000-instead-of-127001), [[2]](http://winhelp2002.mvps.org/hosts.htm), [[3]](https://someonewhocares.org/hosts/))
And `0` can be used as a `0.0.0.0` replacement but this does not work universally. ([source](https://github.com/StevenBlack/hosts#why-not-use-0-instead-of-0000))

In conclusion, I think this is a correctly formatted hosts file (which is compatible with Pi-hole), but as I said in the issue you mentioned, the last update is 3 years and a half old.

**Comment by WaLLy3K** [comment]
Pi-hole will remove everything before the whitespace, as well as the whitespace itself - so what a HOSTS file starts with doesn't matter.

However yes, ReddestDream doesn't update his list any more due to a lack of time. Since it's mostly ad servers, I've been happy for it to stay up.

**Comment by Aeriem** [comment]
Oh, right! I simply forgot to mention the most important bit: it just doesn't matter to Pi-hole (as exemplified by the Disconnect and justdomains lists). xD
Thanks @WaLLy3K!
